src/main/java/com/softserve/ita/java442/cityDonut/repository/CategoryRepository.java

```diff
@@ -0,0 +1,6 @@
+package com.softserve.ita.java442.cityDonut.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface CategoryRepository extends JpaRepository {
```

Comment:
Maybe, it should "extends JpaRepository<Category, Long>"

Reply:
Definitely, JpaRepository is Generic and should extend <Category, Long>